pkg/build/builder/dockerutil.go

```diff
@@ -2,11 +2,21 @@ package builder
 
 import (
 	"os"
+	"time"
 
-	"github.com/fsouza/go-dockerclient"
+	docker "github.com/fsouza/go-dockerclient"
+	"github.com/golang/glog"
 	"github.com/openshift/source-to-image/pkg/tar"
 )
 
+var (
+	// Number of retries of pushing the built Docker image into configured
+	// repository
+	DefaultPushRetryCount = 2
```

Comment:
2 - which results in just *single* retry is not enough IMHO. Either the delay between retries should be closer to 1 minute or we should retry more times. @smarterclayton ?